nilearn/datasets/func.py

```diff
@@ -1669,3 +1669,219 @@ def fetch_cobre(n_subjects=10, data_dir=None, url=None, verbose=1):
+           'A00054533', 'A00054534', 'A00054621', 'A00054895', 'A00054897',
+           'A00054913', 'A00054929', 'A00055061', 'A00055215', 'A00055352',
+           'A00055353', 'A00055542', 'A00055738', 'A00055763', 'A00055806',
+           'A00056097', 'A00056098', 'A00056164', 'A00056372', 'A00056452',
+           'A00056489', 'A00056949']
+
+    nitrc_ids = range(8260, 8470)
+    max_subjects = len(ids)
+    if n_subjects is None:
+        n_subjects = max_subjects
+    if n_subjects > max_subjects:
+        warnings.warn('Warning: there are only %d subjects' % max_subjects)
+        n_subjects = max_subjects
+    ids = ids[:n_subjects]
+    nitrc_ids = nitrc_ids[:n_subjects]
+
+    opts = dict()
+
+    # Dataset description
+    fdescr = _get_dataset_descr(dataset_name)
+
+    # First, get the metadata
+    phenotypic_file = 'NKI_enhanced_surface_phenotypics.csv'
+    phenotypic = (phenotypic_file, url + '8470/pheno_nki_nilearn.csv',
+                  {'move': phenotypic_file})
+
+    phenotypic = _fetch_files(data_dir, [phenotypic], resume=resume,
+                              verbose=verbose)[0]
+
+    # Load the csv file
+    phenotypic = np.genfromtxt(phenotypic, skip_header=True,
+                               names=['Subject', 'Age',
+                                      'Dominant Hand', 'Sex'],
+                               delimiter=',', dtype=['U9', '<f8',
+                                                     'U1', 'U1'])
+
+    # Keep phenotypic information for selected subjects
+    int_ids = np.asarray(ids)
+    phenotypic = phenotypic[[np.where(phenotypic['Subject'] == i)[0][0]
+                             for i in int_ids]]
+
+    # Download subjects' datasets
+    func_right = []
+    func_left = []
+    for i in range(len(ids)):
+
+        archive = url + '%i/%s_%s_preprocessed_fsaverage5_fwhm6.gii'
+        func = os.path.join('%s', '%s_%s_preprocessed_fwhm6.gii')
+        rh = _fetch_files(data_dir,
+                          [(func % (ids[i], ids[i], 'right'),
+                           archive % (nitrc_ids[i], ids[i], 'rh'),
+                           {'move': func % (ids[i], ids[i], 'right')}
+                            )],
+                          resume=resume, verbose=verbose)
+        lh = _fetch_files(data_dir,
+                          [(func % (ids[i], ids[i], 'left'),
+                           archive % (nitrc_ids[i], ids[i], 'lh'),
+                           {'move': func % (ids[i], ids[i], 'left')}
+                            )],
+                          resume=resume, verbose=verbose)
+
+        func_right.append(rh[0])
+        func_left.append(lh[0])
+
+    return Bunch(func_left=func_left, func_right=func_right,
+                 phenotypic=phenotypic,
+                 description=fdescr)
+
+def fetch_surf_fsaverage5(data_dir=None, url=None, resume=True, verbose=1):
+    """ Download Freesurfer fsaverage5 surface
+    Parameters
```

Comment:
circle ci error can be fixed by giving an empty space between one line description and Parameters.